compact_str/src/lib.rs

```diff
@@ -686,3 +681,44 @@ impl fmt::Write for CompactStr {
 }
 
 crate::asserts::assert_size_eq!(CompactStr, String);
+
+/// A trait for converting a value to a `CompactStr`.
+///
+/// This trait is automatically implemented for any type which implements the
+/// [`Display`] trait. As such, `ToCompactStr` shouldn't be implemented directly:
+/// [`Display`] should be implemented instead, and you get the `ToCompactStr`
+/// implementation for free.
+///
+/// [`Display`]: fmt::Display
+pub trait ToCompactStr {
+    /// Converts the given value to a `CompactStr`.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// let i = 5;
+    /// let five = compact_str::CompactStr::from("5");
+    ///
+    /// assert_eq!(five, i.to_string());
+    /// ```
```

Comment:
This example doesn't actually use `.to_compact_str()`?

Reply:
Oops, I simply copied it from the old PR diff.
I will change that.

Reply:
Done.